packages/@react-types/breadcrumbs/src/index.d.ts

```diff
@@ -38,13 +38,6 @@ export interface BreadcrumbItemProps extends PressEvents {
 export interface BreadcrumbsProps<T> {
   /** The breadcrumb items. */
   children: ReactElement<ItemProps<T>> | ReactElement<ItemProps<T>>[],
-  /** Whether the breadcrumbs are used as a heading element. */
-  isHeading?: boolean,
-  /**
-   * Sets the aria-level attribute of the last item, but only if `isHeading` is true.
-   * @default 1
-   */
-  headingAriaLevel?: number,
```

Comment:
I think we need to remove these from `useBreadcrumbItem` too

Reply:
I had considered that previously but decided against it cuz it felt useful for people who wanted to use the hook by itself (kinda like how you used it in the useBreadcrumb documentation). What do you think about that?